### ISSUE TITLE
keyboard focusable?

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#36]
if I understand the purpose of focus correctly, user should be able to move through the photos since they're clickable & necessary to play 

reference: https://usability.yale.edu/web-accessibility/articles/focus-keyboard-operability

**Comment by moates** [comment]
Good news, the photos are focusable in a reasonable order! Bad news...pages that use <<remove>> with nested links get kinda messy.

**Comment by moates** [comment]
Typed elements <<type>> and elements that use <<linkreplace>> or <<remove>> sometimes don't behave as expect wrt focusability. 

When embedding <<type>> in a link, pass the `element 'span'` param so that the focus classes apply inline correctly. Example:
```
<<link "<<type 0ms start 0,5s element 'span'>>Yes please.<</type>>
<</link>>

```


For controlling multiple nested choices, Instead of <<linkreplace>> + <<remove>>, use <<link>> and <<remove>>. I don't totally understand why this fixes the issue, but I suspect it's something under the hood with <<linkreplace>>. Example:
```
    <div id="yesno">    
	<<link "<<type 0ms start 0.5s element 'span'>>Sure, let's share!<</type>>" >>
		  <<replace "#yesno">>
			<<chat 'from-me' 0s>>Sure, let's share!<</chat>>
		<</replace>>
<</link>>
<br>
<<link "<<type 0ms start 0.5s element 'span'>>Nah.<</type>>">>
	<<replace "#yesno">>
    	<<chat 'from-me' 0s>>Nah, sorry.<</chat>>		
        <<chat 'from-them' 30ms>>Okay, suit yourself.<</chat>>
        <</replace>>
<</link>>
</div>
```
